### ISSUE TITLE
Fix addLocalFolder 

### ISSUE THREAD

**Issue by sunloverz** [cthackers/adm-zip#100]
When compressing a folder it creates all folders in the path, but it should add only the last folder
```javascript
var zip = new new AdmZip();
zip.addLocalFolder("/home/xman/projects/lib/../foobar");
```
I think localPath need to path.resolve

**Comment by Elusive138** [comment]
`Utils.findFiles()` returns normalised (`pth.normalize`) paths. This ends up converting drive letters (in Windows) to lower-case.

If you passed in a path with an upper-case drive letter, then the `.replace(localPath, "")` fails because `localPath` with the upper-case letter does not match `path` with the lower-case letter.

@sunloverz's example also applies: the `localPath` param does not match the normalised found paths.

I believe the correct solution is to insert a `localPath = pth.normalize(localPath);` at the beginning of the function. This should cover both @sunloverz and @legionsra's cases. It's also more likely to match actual found files.

Note that @legionsra's suggested split/pop method fails with subdirectories (I think).

**Comment by sunloverz** [comment]
@Elusive138 They are already have merged my pull request https://github.com/cthackers/adm-zip/pulls?q=is%3Apr+is%3Aclosed

**Comment by Elusive138** [comment]
@sunloverz Yea, I just saw. Must be using an older copy here.

**Comment by Elusive138** [comment]
Also, looks like the fixed version was never released (last release, 0.4.4, was 6 months before your fix). That would explain why this bug still occurs in my local copy.
